File: aula5/parte1/java/Chefe.java
public class Chefe extends Personagem {

    private static final int VIDA_FIXA = 200;
    private static final int NIVEL_FIXO = 10;

    private int forca;

    public Chefe(String nome) {
        super(nome, VIDA_FIXA, NIVEL_FIXO);
        this.forca = 20;
    }

    public int getForca() {
        return this.forca;
    }

    @Override
    public String habilidade() {
        return "ataque devastador";
    }

    @Override
    public void ficha() {
        System.out.println("[CHEFE] " + getNome() + " (vida: " + getVida() + ", forca: " + this.forca + ")");
    }
}

File: aula5/parte1/python/chefe.py
from personagem import Personagem


class Chefe(Personagem):
    VIDA_FIXA = 200
    NIVEL_FIXO = 10

    def __init__(self, nome: str):
        super().__init__(nome, Chefe.VIDA_FIXA, Chefe.NIVEL_FIXO)
        self.forca = 20

    def habilidade(self) -> str:
        return "ataque devastador"

    def ficha(self) -> None:
        print(f"[CHEFE] {self.nome} (vida: {self.vida}, forca: {self.forca})")
